src/rt/test/func/pollers/pollers.cc

```diff
@@ -0,0 +1,48 @@
+// Copyright Microsoft and Project Verona Contributors.
+// SPDX-License-Identifier: MIT
+
+#include <ctime>
+#include <test/harness.h>
+
+struct TimerPoller : public VCown<TimerPoller>
+{
+  std::time_t start;
+  bool active;
+
+  TimerPoller() : start(std::time(nullptr)), active(false) {}
+
+  void notified(Object* o)
+  {
+    TimerPoller* p = (TimerPoller*)o;
+    if ((std::difftime(std::time(nullptr), p->start) > 1) && (p->active))
+    {
+      auto& sched = Scheduler::get();
+      sched.poller_remove(p);
+      p->active = false;
+      Cown::release(ThreadAlloc::get(), p);
```

Comment:
This example keeps the poller live, and then performs its own deallocation.  I think you need to add some systematic testing runs to the harness for this code.  I am concerned about the ordering of the `poller_remove` and the `release`.  

Can the following happen
  1. poller_remove is send to a token cown.
  2. p is released, and deallocated
  3. token cown performs notification of a deallocated cown
  4. poller_remove is executed by token cown.